db/re/item_db_usable.yml

```diff
@@ -47035,7 +47035,7 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      rentitem 12212,432000;
+      rentitem 23023,432000;
```

Comment:
This box should give a rental fly wing for 5d (id 12887 I guess) according to kRO description.
Also our english item name is currently wrong.

Reply:
Thanks I will check and fix later.